examples/model_selection/plot_learning_curve.py

```diff
@@ -150,6 +157,10 @@ def plot_learning_curve(estimator, title, X, y, axes=None, ylim=None, cv=None,
 cv = ShuffleSplit(n_splits=100, test_size=0.2, random_state=0)
 
 estimator = GaussianNB()
+# We will use the default GaussianNB and SVC scorers ('mean accuracy'), but
+# others could be also used.
+# You can check other scorers in the model evaluation documentation in
+# https://scikit-learn.org/stable/modules/model_evaluation.html
 plot_learning_curve(estimator, title, X, y, axes=axes[:, 0], ylim=(0.7, 1.01),
                     cv=cv, n_jobs=4)
```

Comment:
I suggest passing `scoring` anyways and remove the comment above. This enables us to use the new `scoring` parameter in the example and makes it explicit that the metric is accuracy.

```suggestion
plot_learning_curve(estimator, title, X, y, axes=axes[:, 0], ylim=(0.7, 1.01),
                    cv=cv, n_jobs=4, scoring="accuracy")
```